sgx/gramine/CI-Examples/generate-token/python.manifest.template

```diff
@@ -10,9 +10,9 @@ loader.pal_internal_mem_size = "200M"
ncher-2.7.1.jar:/opt/meituan/hadoop/share/hadoop/yarn/hadoop-yarn-registry-2.7.1.jar:/opt/meituan/hadoop/share/hadoop/yarn/hadoop-yarn-applications-distributedshell-2.7.1.jar:/opt/meituan/hadoop/share/hadoop/yarn/hadoop-yarn-api-2.7.1.jar:/opt/meituan/hadoop/share/hadoop/yarn/hadoop-yarn-server-nodemanager-2.7.1.jar:/opt/meituan/hadoop/share/hadoop/yarn/hadoop-yarn-server-applicationhistoryservice-2.7.1.jar:/opt/meituan/hadoop/share/hadoop/mapreduce/lib/protobuf-java-2.5.0.jar:/opt/meituan/hadoop/share/hadoop/mapreduce/lib/commons-compress-1.4.1.jar:/opt/meituan/hadoop/share/hadoop/mapreduce/lib/xz-1.0.jar:/opt/meituan/hadoop/share/hadoop/mapreduce/lib/avro-1.7.4.jar:/opt/meituan/hadoop/share/hadoop/mapreduce/lib/jersey-guice-1.9.jar:/opt/meituan/hadoop/share/hadoop/mapreduce/lib/aopalliance-1.0.jar:/opt/meituan/hadoop/share/hadoop/mapreduce/lib/asm-3.2.jar:/opt/meituan/hadoop/share/hadoop/mapreduce/lib/hamcrest-core-1.3.jar:/opt/meituan/hadoop/share/hadoop/mapreduce/lib/leveldbjni-all-1.8.jar:/opt/meituan/hadoop/share/hadoop/mapreduce/lib/jersey-core-1.9.jar:/opt/meituan/hadoop/share/hadoop/mapreduce/lib/jackson-mapper-asl-1.9.13.jar:/opt/meituan/hadoop/share/hadoop/mapreduce/lib/commons-io-2.4.jar:/opt/meituan/hadoop/share/hadoop/mapreduce/lib/hadoop-annotations-2.7.1.jar:/opt/meituan/hadoop/share/hadoop/mapreduce/lib/jackson-core-asl-1.9.13.jar:/opt/meituan/hadoop/share/hadoop/mapreduce/lib/junit-4.11.jar:/opt/meituan/hadoop/share/hadoop/mapreduce/lib/javax.inject-1.jar:/opt/meituan/hadoop/share/hadoop/mapreduce/lib/guice-3.0.jar:/opt/meituan/hadoop/share/hadoop/mapreduce/lib/snappy-java-1.0.4.1.jar:/opt/meituan/hadoop/share/hadoop/mapreduce/lib/hadoop-lzo-0.4.20-SNAPSHOT.jar:/opt/meituan/hadoop/share/hadoop/mapreduce/lib/log4j-1.2.17.jar:/opt/meituan/hadoop/share/hadoop/mapreduce/lib/guice-servlet-3.0.jar:/opt/meituan/hadoop/share/hadoop/mapreduce/lib/paranamer-2.3.jar:/opt/meituan/hadoop/share/hadoop/mapreduce/lib/netty-3.6.2.Final.jar:/opt/meituan/hadoop/share/hadoop/mapreduce/lib/jersey-server-1.9.jar:/opt/meituan/hadoop/share/hadoop/mapreduce/hadoop-mapreduce-client-jobclient-2.7.1.jar:/opt/meituan/hadoop/share/hadoop/mapreduce/hadoop-mapreduce-client-hs-plugins-2.7.1.jar:/opt/meituan/hadoop/share/hadoop/mapreduce/hadoop-mapreduce-client-jobclient-2.7.1-tests.jar:/opt/meituan/hadoop/share/hadoop/mapreduce/hadoop-mapreduce-client-app-2.7.1.jar:/opt/meituan/hadoop/share/hadoop/mapreduce/hadoop-mapreduce-client-common-2.7.1.jar:/opt/meituan/hadoop/share/hadoop/mapreduce/hadoop-mapreduce-client-core-2.7.1.jar:/opt/meituan/hadoop/share/hadoop/mapreduce/hadoop-mapreduce-client-shuffle-2.7.1.jar:/opt/meituan/hadoop/share/hadoop/mapreduce/hadoop-mapreduce-examples-2.7.1.jar:/opt/meituan/hadoop/share/hadoop/mapreduce/hadoop-mapreduce-client-hs-2.7.1.jar:/opt/meituan/hadoop/contrib/capacity-scheduler/*.jar:/opt/meituan/hadoop/contrib/capacity-scheduler/*.jar:/opt/tiger/jdk/openjdk-1.8.0_265/lib/dt.jar:/opt/tiger/jdk/openjdk-1.8.0_265/lib/tools.jar"
```

Comment:
remove `loader.env.CLASSPATH`, it's loaded from host now.

Reply:
OK，Already removed